### DISCUSSION TITLE
Not able to authenticate the Microsoft docs organization

### DISCUSSION THREAD

**Discussion by ghost** [community/community#62570] [New to GitHub]
I am not able to authenticate the Microsoft docs organization invitation. I see below error. @PriyankaSinha07 is linked with my MS ID prsinha. I cannot unlink my MS account from @PriyankaSinha-Dynamics365's account as this is old account and I dont remember the PWD or recovery.

**There was an issue joining the enterprise: Your GitHub user account @PriyankaSinha07 is currently unlinked. However, you are attempting to authenticate with your Identity Provider using the 'prsinha@microsoft.com' SAML identity which is already linked to a different GitHub user account in the organization. Please reach out to one of your GitHub organization owners for assistance. and External ID '68167e8b-9569-43e5-b957-53628d6a718c' is already linked to @PriyankaSinha-Dynamics365's account.**

**Comment by queenofcorgis** [comment]
Hi there - have you contacted your organization's owners as stated? I'd follow up with them first, then I'd recommend opening a ticket on our [Support page](https://support.github.com/contact) as this is likely involving account-specifics from your two accounts. Best of luck!

**Comment by akorp** [comment]
An org owner one can go to `https://github.com/orgs/<your_org>/people/<user_account>/sso` to unlink the old account.
